### PR TITLE
Issue #13345: Enable examples tests for SimplifyBooleanExpressionCheck

### PR DESCRIPTION
**Pull request by AmitKumarDeoghoria**
### Title: Enable examples tests for SimplifyBooleanExpressionCheck

**Related Issue: #13345** 

**Summary**
This pull request aims to enable example tests for the **SimplifyBooleanExpressionCheck** in the project. The implementation focuses on ensuring that the SimplifyBooleanExpression checks are correctly validated through comprehensive test cases, improving overall code reliability and quality.

**Key Changes Made**

- Conversion of txt file to Java File
- Enable the test cases by adding expected violations in the expected array for all example tests
- Made corresponding changes in the xml and xml.template file

### CODE REVIEW

**Comment by AmitKumarDeoghoria** [comment]
GitHub, generate website

**Comment by AmitKumarDeoghoria** [comment]
GitHub, generate website

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/simplifybooleanexpression/Example1.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    if (!false) {}; // violation, can be simplified to true
+    if (a == true) {}; // violation, can be simplified to a
+    if (a == b) {};
+    if (a == false) {}; // violation, can be simplified to !a
+    if (!(a != true)) {}; // violation, can be simplified to a
+    e = (a || b) ? c : d;
+    e = (a || false) ? c : d; // violation, can be simplified to a
+    e = (a && b) ? c : d;
+    int s = 12;
+    boolean m = s > 1 ? true : false; // violation, can be simplified to s > 1
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/simplifybooleanexpression/Example1.java:27]: it is ok to use explanation comment and not a message.
as it explains what is expected code and not just pointing on line with violation

**Review by romani — APPROVED** [review]
ok to merge.

**Pull request merged**